HW_3/20.py

```diff
@@ -0,0 +1,5 @@
+w = input("Введите слово: ")
+word = str.upper(w)
+print(word)
+dct = {1:"AEIOULNSTRАВЕИНОРСТ", 2:"DGДКЛМПУ", 3:"BCMPБГЁЬЯ", 4:"FHVWYЙЫ", 5:"KЖЗХЦЧ", 8:"JXШЭЮ", 10:"QZФЩЪ"}
+print(sum([i for letter in word for i, val in dct.items() if letter in val]))
```

Comment:
Каскадный генератор! Круто. Не каждый новичок сможет написать каскадный генератор.
```suggestion
print(sum(i for letter in word for i, val in dct.items() if letter in val))
```
Однако, если переписать словарь то в каскадном генераторе не будет необходимости.